shared/app/tab-bar/container.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import {connect} from 'react-redux'
+import TabBarRender from './index.render'
+
+import type {TypedState} from '../../constants/reducer'
+
+const mapStateToProps = (state: TypedState) => ({
```

Comment:
Should we create a selector for this? otherwise this returns a new object and could cause rerenders